packages/compiler/src/plugins.ts

```diff
@@ -77,6 +78,27 @@ export const processMyst = () => {
           };
           (parent as Parent).children[index] = newNode;
           return index;
+        } else if (mystType === "panels") {
+          let cards = parsePanels(value);
+          console.log(cards);
+          // parse each card
+          const newNode = {
+            type: "html",
+            value: mustache.render(
+              `<Panels>
+               <div>
+               {{#cards}}
+               {{header}}
```

Comment:
Since there are HTML, we will want to use the `{{{` syntax so they are not HTML escaped. See https://mustache.github.io/mustache.5.html and search for `{{{`

You can see an example of this here: https://github.com/irydium/irydium/blob/1e7c42a55bc0632ef5bf841176fd8b10bb280893/packages/compiler/src/templates/tasks.js#L13

Reply:
Was able to use this with the panels mustache template, even including conditionals.